game_gui.rb

```diff
@@ -94,6 +95,20 @@ def create_card_images(deck)
         return result
     end
 
+    def create_tool_tip_images(deck)
+        result = {}
+        deck.each do |card|
+            if card.respond_to?(:rule_text)
+                result[card.name] = GuiUtil.get_text_image(card.rule_text, @font)
+            elsif card.card_type == "Keeper" && card.is_peace?
+                result[card.name] = GuiUtil.get_text_image("#{card.name}\n (If you have War, move it to another player if you have this on the table)", @font)
+            else
+                result[card.name] = GuiUtil.get_text_image(card.name, @font)
+            end
```

Comment:
Perhaps I should consider building `rule_text` into every card. That way this isn't an issue. The main problem is I am not sure how to do "rules text" for keepers.